### PR TITLE
CLDR-17366 kbd: rename `unicodeSet` to `uset`

### PR DESCRIPTION
**Pull request by srl295**
CLDR-17366

- [X] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: docs/ldml/tr35-keyboards.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ The LDML specification is divided into the following parts:
   * [Element: variables](#element-variables)
   * [Element: string](#element-string)
   * [Element: set](#element-set)
-  * [Element: unicodeSet](#element-unicodeset)
+  * [Element: uSet](#element-uset)
```
Comment (mcdurdin) [inline docs/ldml/tr35-keyboards.md:97]: Can we go with lower case? It's gonna be kinda ugly for keyboard devs with this mixed case. It's not really two words any more...

```suggestion
  * [Element: uset](#element-uset)
```

File: docs/ldml/tr35-keyboards.md [hunk 2]
```diff
@@ -1953,11 +1954,11 @@ The following are additions to standard Regex syntax.
 
     In this usage, the variable with `id="zwnj"` will be substituted in at this point in the expression. The variable can contain a range, a character, or any other portion of a pattern. If `zwnj` is a simple string, the pattern will match that string at this point.
 
-- **Set and UnicodeSet variables**
+- **Set or uSet variables**
```
Comment (mcdurdin) [inline docs/ldml/tr35-keyboards.md:1957]: If we go with lower casing, then:

```suggestion
- **Set or uset variables**
```

or

```suggestion
- **Uset or set variables**
```

**Review by mcdurdin — CHANGES_REQUESTED** [review]

File: docs/ldml/tr35-keyboards.md (hunk 2 above)
Comment (srl295) [reply docs/ldml/tr35-keyboards.md:1957]: I'm going to go with lowercase but monospace

File: keyboards/dtd/ldmlKeyboard3.dtd [hunk 3]
```diff
@@ -171,7 +171,7 @@ Please view the subcommittee page for the most recent information.
     <!--@MATCH:any-->
     <!--@VALUE-->
 
-<!ELEMENT variables ( import*, string*, set*, unicodeSet*, special* ) >
+<!ELEMENT variables ( import*, string*, set*, uSet*, special* ) >
```
Comment (macchiati) [inline keyboards/dtd/ldmlKeyboard3.dtd:174]: You are using "uset" in some places and "uSet" in others. The uSet cases need to be changed. Recommend global case-sensitive search.

**Review by macchiati — CHANGES_REQUESTED** [review]

File: keyboards/dtd/ldmlKeyboard3.dtd (hunk 3 above)
Comment (srl295) [reply keyboards/dtd/ldmlKeyboard3.dtd]: fixed.

**Review by macchiati — APPROVED** [review]

**Pull request merged**